XPathParser/project.json

```diff
@@ -11,10 +11,11 @@
     "repository": {
       "type": "git",
       "url": "https://github.com/quamotion/XPathParser"
-    },
+    }
   },
 
   "dependencies": {
+    "NETStandard.Library": "1.6.0"
```

Comment:
No worries. It's a good opportunity for me to get more familiar with the project.json stuff - especially before it goes away again :-)